frontend/tet/admin/Dockerfile

```diff
@@ -26,6 +26,8 @@ COPY --chown=appuser:appuser . .
 FROM appbase as development
 # =============================
 
+ENV NODE_OPTIONS="--max_old_space_size=8192"
```

Comment:
This was added because Docker Compose was crashing due to V8 heap out of memory (both I and @pekkvil experienced this problem). We should still check that this doesn't cause the container to use more memory in review and other environments.